salt/orch/etcd-migrate.sls

```diff
@@ -46,7 +48,7 @@ backup-etcd:
     - tgt_type: grain_pcre
     - name: cmd.run
     - arg:
-      - mkdir /tmp/backup; btrfs subvolume snapshot /var/lib/etcd /tmp/backup/etcd
+      - if [ -d /var/lib/etcd/member ] && ! [ -d /tmp/backup ]; then mkdir /tmp/backup; btrfs subvolume snapshot /var/lib/etcd /tmp/backup/etcd; fi
```

Comment:
I wonder should we put a datestamp into the folder name? to avoid repeated attempts blowing away the original backup.

Reply:
i guess the `! [ -d /tmp/backup ]` condition will prevent this

Reply:
I've added `date` timestamp